layout/theme.liquid

```diff
@@ -132,15 +132,30 @@
     {% include 'cart-popup' %}
   {%- endif -%}
 
-  {% section 'header' %}
+
+{% comment %}======= Hiding Header from cart pagewith conditional {% endcomment %}
+
+  {% unless template.suffix contains "cart" %}
+          {% section 'header' %}
+     {% endunless %}
+
+{% comment %}=================================================== {% endcomment %}
+
 
   <div class="page-container drawer-page-content" id="PageContainer">
 
     <main class="main-content js-focus-hidden" id="MainContent" role="main" tabindex="-1">
       {{ content_for_layout }}
     </main>
 
-    {% section 'footer' %}
+{% comment %}======= Hiding Footer from cart pagewith conditional {% endcomment %}
+
+{% unless template.suffix contains "cart" %}
```

Comment:
Most likely not working due to template file name being cart.liquid (no suffix, i.e. page.suffix.liquid).  Change Conditional.